docs/fast_data/faq/benchmark_fast_data.md

```diff
@@ -0,0 +1,155 @@
+:::
+
+## Assumptions
+
+This table shows details for each component that has been monitored and analyze to provide the guidelines.
+
+<table>
+    <tr><th>Component</th><th>Replicas</th><th>Details</th></tr>
+    <tr><td>Kafka</td><td>4 brokers</td><td> 600.000 nominal events X day, i.e. 30GB of ingestion messages.</td></tr>
+    <tr><td>MongoDB</td><td>3 replica sets</td><td>Enterprise distribution having 32 GB of RAM.</td></tr>
+    <tr><td>Fast Data</td><td>6 nodes</td><td>K8S cluster with each node having 8 core.</td></tr>
+</table>
+
+
+## Kafka
+
+
+### What is the topic setup I should use to guarantee an high rate of ingestion?
+
+
+When creating and managing topics in a Kafka cluster, is crucial to define a number of partitions that can balance the identifiers of messages. This means that a topic:
+
+* linked to a data source having a fixed cardinality, requires a number of partitions `<=` 3;   
+* linked to a data source having an higher number of insert/update/delete operation requires a number of partition `>` 3.
+
+:::tip Use Case
+A topic having an ingestion rate of `2.2k message/minute` requires 10 partitions to distribute the messages among the topics while speeding up micro-services performances.
+:::
+
+### What is the topic setup I should use to guarantee an high availability?
+
+A topic is distributed among the brokers of your cluster, with a parameter called _replication factor_: for critical messages, this value should be set to the number of brokers, meaning that the data of a topic will be stored inside all brokers.
+
+You can also specify how many replicas are needed to send acknowledgments on produced messages before committing, to guarantee the synchronization of the distributed data. This value can be set at the topic level with the parameter `min.insync.replicas` and is usually equal to `# of brokers - 1`. Critical flows may require a number of in-sync replicas equal to the number of brokers. 
+
+## Mongo Db
+
+### What should be the required resources to perform an Initial Load?
+
+The **Initial Load** is a manual operation made at runtime to initialize the projection's records with the most recent copy of the dataset from the system of record linked to it.
+
+This activity is performed through the **Change Data Capture** (**CDC**) and will be performed by the [Real-Time Updater (RTU)](/fast_data/realtime_updater.md).
+
+:::danger Before executing the Initial Load...
+Since an high number of ingestion messages will be handled by the RTU, remember **to disable strategy** during the execution of the Initial Load.
+
+This will avoid to perform additional MongoDb queries made by the strategies to generate projection changes records.
+:::
+
+:::tip Execution Time
+An initial load of `6.100.000 records` took around _10 minutes_, which is approximately a rate of `58k message/minute`, using up until 70% of CPU from a single MongoDb core.  
```

Comment:
Great accuracy, but if we're talking about approximate numbers, let's also round the values of records. Or if we have the exact messages/minute, let's rephrase to leverage the precision of that data and then hypothesize the records over time instead.